src/coreComponents/common/Path.hpp

```diff
@@ -77,17 +77,21 @@ class Path : public std::string
 /*!
  * @brief Gets the absolute path of a file
  * @param[in] path the relative path to the file
- * @param[out] absolutePath the absolute path to the file
+ * @return the absolute path to the file resolved from current working directory
+ * @exception InputError if absolute path could not be resolved
  */
-void getAbsolutePath( std::string const & path, std::string & absolutePath );
+std::string getAbsolutePath( std::string const & path );
```

Comment:
Re: `string` vs `std::string`. Here we would be passing `string` as the argument, so it should have a UDC to `std::string`, or be an alias for `std::string`.